Dockerfile

```diff
@@ -0,0 +1,17 @@
+#stage 1. Build Executable binary
+FROM golang:alpine as builder
+
+RUN apk update && apk add --no-cache git
+
+WORKDIR $GOPATH/src/mypackage/myapp
+COPY . .
+
+# get all dependencies
+RUN go get -d -v
+
+#build binary
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o /go/bin/rest_bomber
+
+#Stage 2. Build small version of image
```

Comment:
hmm, It seems we have to add second FROM or maybe I don't know something

Reply:
yes, you're right. I fix it